client/extensions/woocommerce/components/product-image-uploader/index.js

```diff
@@ -202,13 +202,16 @@ class ProductImageUploader extends Component {
 
 	renderUploader() {
 		const { translate, multiple } = this.props;
+
+		const addString = multiple ? translate( 'Add images' ) : translate( 'Add image' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Add Image' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-product-cat-form-fields)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).